src/components/Canvas/Sections/FontFamilies.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="space-y-6">
+    <div v-for="[prop, value] in fontFamilies" :key="prop">
+      <p
+        class="mb-2 leading-none text-2xl text-gray-900 dark:text-gray-500"
+        :style="{
+          fontFamily: getFontFamilyValue(value)
+        }"
+      >
+        {{ data.typographyExample }}
+      </p>
+      <CanvasBlockLabel :label="`font-${prop}`" :value="getFontFamilyValue(value)" />
+    </div>
+  </div>
+</template>
+
+<script>
+import CanvasBlockLabel from '../CanvasBlockLabel'
+
+export default {
+  components: {
+    CanvasBlockLabel
+  },
+  props: {
+    data: {
+      type: Object,
+      required: true
+    },
+    config: {
+      type: Object,
+      required: true
+    }
+  },
+  computed: {
+    fontFamilies () {
+      return Object.entries(this.data.fontFamily)
+    }
+  },
+  methods: {
+    getFontFamilyValue (value) {
+      if (Array.isArray(value)) {
+        // arbitrary length
+        return value.slice(0, 9).join(', ')
```

Comment:
@acharlop Why do we need the `.slice` here?

Reply:
the default tailwind font of `font-sans` based off of [modern-normalize](https://github.com/sindresorhus/modern-normalize/blob/main/modern-normalize.css#L43) has a value of:

```
system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"
```

I figured it's a bit overkill to show and instead up to `sans-serif`, but open to changing it if you have a different preference, or add a `...` after

<img width="1350" alt="image" src="https://user-images.githubusercontent.com/8885448/165258796-e20a8fbb-fd85-4841-8ade-4b9f47ef900c.png">